torch/nn/parallel/distributed.py

```diff
@@ -315,6 +315,19 @@ def _ddp_init_helper(self):
         (5) passing a handle of DDP to SyncBatchNorm Layer
         """
         if self.device_ids and len(self.device_ids) > 1:
+
+            import warnings
+            warnings.warn(
+                "Single-Process Multi-GPU is not the recommended mode for "
```

Comment:
This message seems reasonable, but isn't there *also* a known issue that raises an error, i.e. https://github.com/pytorch/pytorch/issues/36268?  This message just talks about performance.

Reply:
Updated the warning message.